templates/new_request_page.hbs

```diff
@@ -41,12 +41,21 @@
     organizations: {{json user.organizations}},
     wysiwyg: true,
     answerBotModal: {
-      answerBot: {{json answer_bot}},
+      answerBot: {{#if answer_bot}}{{json answer_bot}}{{else}}null{{/if}},
+      answerBotGenerative: {{#if answer_bot_generative}}{{json answer_bot_generative}}{{else}}null{{/if}},
```

Comment:
why do we need to validate if its set here? it didn't break before, why do we need to protect now? 🤔